docs/registry/README.md

```diff
@@ -0,0 +1,193 @@
+# Registry Proposal
+
+## Objective
+
+Design an **initial** version of the Registry that can support discoverability of 
+the different event types that can be consumed from the eventing mesh.
+
+## Requirements
+
+Our design revolves around the following core requirements:
+
+1. We should have a Registry per namespace to enforce isolation.
```

Comment:
Registry is used to manage EventType. I don't quite understand why EventType shall be isolated by namespace. I think it's good to isolate event sources by namespace. Yet EventType supported shall be the capability of a platform.